tools/provision-alerts/template/template.yml

```diff
@@ -13,6 +13,33 @@ policies:
           FROM {{ .Sample }}
           WHERE displayName IN ('{{ .DisplayNameCurrent }}','{{ .DisplayNamePrevious }}')
 
+      - name: Process count comparator
+        nrql:
```

Comment:
wouldn't this alert be triggered often by different processes that start automatically and die? for e.g. on windows, the windows update checker

Reply:
I guess it would need to be a process that lives for more than 10 minutes. Right now the difference in 24h is 0 but we could increase the threshold to ~5 to be safer
nrql example:
```
SELECT abs( filter(uniqueCount(commandLine),WHERE displayName like 'canary:v1.20.7:amd64:windows_2019') - filter(uniqueCount(commandLine),WHERE displayName like 'canary:v1.21.0:amd64:windows_2019') ) FROM SystemSample WHERE displayName IN ('canary:v1.20.7:amd64:windows_2019','canary:v1.21.0:amd64:windows_2019') TIMESERIES 10 minute SINCE 1 day ago
```